### ISSUE TITLE
Signal Catching

### ISSUE THREAD

**Issue by victorstewart** [PlatformLab/NanoLog#44]
do you plan to incorporate signal catching into NanoLog? Would be great to have them automatically caught and logged rather than having to deduce what went wrong, then instrument with logging, then rerun... rinse repeat. And/or to avoid having to use another signal logging library.

**Comment by syang0** [comment]
No, I currently do not have any plans to incorporate signal catching NanoLog. However if you’d like to add it and issue a pull request, I’d love to take it in :)
